cmake/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(cmake)
+
+set(CMAKE_CXX_STANDART 14)
+
+add_executable(hello main.cpp lib/hello_world.cpp lib/hello_world.h)
```

Comment:
Why are you compiling Header files with .cpp?
You just need to set Include dir location.

Reply:
Fixed (but im not sure)